include/ensmallen_bits/callbacks/early_stop_at_min_loss.hpp

```diff
@@ -76,6 +112,9 @@ class EarlyStopAtMinLoss
 
   //! Locally-stored number of steps since the loss improved.
   size_t steps;
+
+  //! function to call at the end of the epoch
```

Comment:
```suggestion
  //! Function to call at the end of the epoch.
```

Start with a capital letter and use a `.` at the end.